### PR TITLE
24-use-redis-as-rails-cache-store-in-production

### PR DESCRIPTION
**Pull request by StupidCodeFactory**
https://trello.com/c/ORi61T9d/24-use-redis-as-rails-cache-store-in-production

## Description
Add a shared cache, it uses the Sidekiq one.

## Checklist:

- [ ] Automated checks are passing locally.
- [ ] CHANGELOG updated if change is worth telling users about.
### General testing
- [ ] Test without javascript
- [ ] Test on small screen
### Accessibility testing
- [ ] Works keyboard only
- [ ] Tested with one screen reader
- [ ] Zoom page to 400% - content still visible
- [ ] Disable css - does content make sense and appear in a logical order?
- [ ] Passes automated checker (automated in build or manual)

### CODE REVIEW

File: infrastructure/ci/build-docker-images.sh [hunk 1]
```diff
@@ -13,7 +13,7 @@ function docker_tag_exists {
 }
 
 echo "Logging into DockerHub"
-docker login -u $DOCKER_USERNAME -p $DOCKER_PASSWORD
+echo $DOCKER_PASSWORD docker login -u $DOCKER_USERNAME --password-stdin
```
Comment (slorek) [inline infrastructure/ci/build-docker-images.sh]: Won't this break the build as this script is run by Travis?

File: psd-web/config/environments/production.rb [hunk 2]
```diff
@@ -49,7 +49,9 @@
   config.log_tags = [:request_id]
 
   # Use a different cache store in production.
-  config.cache_store = :memory_store
+  config.cache_store = :redis_cache_store, {
+    url: Rails.application.config_for(:redis_queue)
```
Comment (slorek) [inline psd-web/config/environments/production.rb]: I think we should rename `redis_queue` to `redis_queue_and_cache_store` or just `redis_store`.

File: infrastructure/ci/build-docker-images.sh (hunk 1 above)
Comment (StupidCodeFactory) [reply infrastructure/ci/build-docker-images.sh]: I doubt that's it -but that does not look right though it's missing a `|`

Comment (StupidCodeFactory) [reply infrastructure/ci/build-docker-images.sh]: yes that is it 🤦‍♂

**Pull request closed without merging**